edge/pkg/edged/edged_pods.go

```diff
@@ -69,8 +70,10 @@ import (
 	"k8s.io/kubernetes/third_party/forked/golang/expansion"
 	utilfile "k8s.io/utils/path"
 
+	"github.com/kubeedge/kubeedge/common/constants"
 	edgedconfig "github.com/kubeedge/kubeedge/edge/pkg/edged/config"
 	pkgutil "github.com/kubeedge/kubeedge/pkg/util"
+	netutil "k8s.io/apimachinery/pkg/util/net"
```

Comment:
this import package need to be moved to the second part, together with the below packages
```
"k8s.io/kubernetes/third_party/forked/golang/expansion"
	utilfile "k8s.io/utils/path"
```
or, will not pass the gci lint checks

Reply:
ok, will fix this